src/platform/emotes/data.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  generateValidator,
+  JSONSchema,
+  ValidateFunction
+} from '../../validation'
+
+/** @alpha */
+export type EmoteData = {
+  loop: boolean
+  thumbnail: string
+}
+
+/** @alpha */
+export namespace EmoteData {
+  export const schema: JSONSchema<EmoteData> = {
+    type: 'object',
+    properties: {
+      loop: {
+        type: 'boolean'
+      },
+      thumbnail: {
```

Comment:
Where is thumbnail stored? 
Is it an URL?
Is it a content file reference?

Reply:
Is the same as this type of data:
https://github.com/decentraland/common-schemas/blob/10a57e35c94940f6c6a66ef45d102e2099be946a/src/platform/wearables/wearable.ts#L30

Reply:
Could you add a code comment to explain what the field is and how it works?